build.gradle

```diff
@@ -45,8 +46,9 @@ subprojects {
     dependencies {
         implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
-        testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
-        testImplementation "junit:junit:4.12"
+        testCompile "org.junit.jupiter:junit-jupiter-api:${junit_platform_version}"
+        testCompile "org.junit.jupiter:junit-jupiter-params:${junit_platform_version}"
+        testRuntime "org.junit.jupiter:junit-jupiter-engine:${junit_platform_version}"
```

Comment:
Could you please double check this with:
https://junit.org/junit5/docs/current/user-guide/#running-tests-build-gradle

and 

https://github.com/junit-team/junit5-samples/blob/r5.6.0/junit5-jupiter-starter-gradle-kotlin/build.gradle.kts

If the dependencies are really needed.

Reply:
Fixed. Done acoording to: https://junit.org/junit5/docs/current/user-guide/#running-tests-build-gradle-engines-configure